bindings/netstandard/ElectionGuard/ElectionGuard.Encryption.Tests/TestEncrypt.cs

```diff
@@ -2,7 +2,7 @@
 using ElectionGuard.Encryption.Utils;
 using NUnit.Framework;
 
-namespace ElectionGuard.Encrypt.Tests
+namespace ElectionGuard.Encryption.Tests
```

Comment:
I would eventually like to change all namespaces, but for now just boy scouting the files I change